examples/template-common.js

```diff
@@ -33,6 +33,7 @@ exports.replaceBase = (template) => {
 	return template
 		.replace(/\r\n/g, "\n")
 		.replace(/\r/g, "\n")
+		.replace(/\[webpack-cli\].+\n/g, "")
```

Comment:
This should be removed in webpack-cli

Reply:
Other `plugins` and `loaders` can generate `warnings` and `errors` too. Hence we added `[webpack-cli]` to have a better distinction between errors & warnings. (If I remember correctly)

Reply:
Sorry that was a bit unclear. I'm not against the prefix. I was only complaining about the `Compilation finished` message, which is redundant to the `webpack compiled successfully` message in the stats output.

A prefix is good in general for all other messages...